pkg/reconciler/errors.go

```diff
@@ -1,4 +1,4 @@
-package main
+package reconciler
```

Comment:
not sure where to put this comment: now that you've dropped the reconciler cmd, you can also remove this file. None of these errors are used now.